cms/envs/bok_choy.env.json

```diff
@@ -74,7 +74,6 @@
         "SUBDOMAIN_BRANDING": false,
         "SUBDOMAIN_COURSE_LISTINGS": false,
         "ALLOW_ALL_ADVANCED_COMPONENTS": true,
-        "ALLOW_COURSE_RERUNS": true,
```

Comment:
@andy-armstrong is this how you tried enabling student notes for bok choy tests?